lib/AST/FineGrainedDependencyFormat.cpp

```diff
@@ -322,7 +323,11 @@ class Serializer {
   Serializer(llvm::BitstreamWriter &ExistingOut) : Out(ExistingOut) {}
 
 public:
-  void writeFineGrainedDependencyGraph(const SourceFileDepGraph &g);
+  /// If \param forSwiftDeps is \c true, the resulting file will contain
+  /// a leading magic signature and bitstream metadata. Otherwise, this
+  /// information is omitted.
```

Comment:
As I wrote above, you could rename the parameter to "includeWrapping" or "includeSignatureAndMetadata".
Then the comment is not needed. (just a nit)